### PR TITLE
scroll to top

### PR DESCRIPTION
**Pull request by MaysShtanwi**
scroll to top

### CODE REVIEW

File: css/custom.css [hunk 1]
```diff
@@ -7,6 +7,26 @@ body {
 * {
     box-sizing: border-box;
 }
+.toTop{
```
Comment (ngTurk) [inline css/custom.css]: space after `.toTop`

File: css/custom.css [hunk 2]
```diff
@@ -7,6 +7,26 @@ body {
...
+background-color: #00b3b3;
```
Comment (ngTurk) [inline css/custom.css]: styles ==> alphabetical sort and check indentation

File: js/index.js [hunk 3]
```diff
@@ -34,3 +37,12 @@ document.addEventListener("scroll", () => {
...
 });
+
+//scroll to top
+$scrollButton.addEventListener('click', () => {
+  window.scrollTo({
+    top: 0,
+    left: 0,
+    behavior: 'smooth'
+  });
+});
```
Comment (ngTurk) [inline js/index.js]: add line in the end of the file

**Review by ngTurk — CHANGES_REQUESTED** [review]

File: css/custom.css (hunk 1 above)
Comment (MaysShtanwi) [reply css/custom.css]: very well

File: css/custom.css (hunk 2 above)
Comment (MaysShtanwi) [reply css/custom.css]: yes sir

File: js/index.js (hunk 3 above)
Comment (MaysShtanwi) [reply js/index.js]: got it!

File: css/custom.css (hunk 1 above)
Comment (ngTurk) [reply css/custom.css]: change this to `.to-top`

**Review by ngTurk — CHANGES_REQUESTED** [review]

**Review by ngTurk — APPROVED** [review]

**Pull request merged**